CI/travis/check_kernel.sh

```diff
@@ -59,10 +59,10 @@ echo
 sed -i '/IIO_NO_MOD/d' /tmp/libiio_iio_modifier
 
 while IFS="" read -r p ; do
-	key=$(echo ${p//[[:space:],]/})
+	key=$(echo "${p//[[:space:],]/}")
```

Comment:
my shell-fu is a bit vague;
especially with bash;
question is: does this still evaluate the same here after adding quotes?

fwiw/generally: i also tried a while back to do this shellcheck fixing of double-quotes, but quite often i would need some vars to evaluate without the quotes there;

the semantic with/without double-quotes is a bit too subtle sometimes, and may not always be worth the risk of fixing
that is: unless we re-test everything

Reply:
I tried this by hand - and yes - it evaluates to the same.